drake/core/test/testVector.cpp

```diff
@@ -1,9 +1,18 @@
 
-#include "drake/examples/Pendulum/Pendulum.h"  // to get some types
-#include "drake/util/testUtil.h"
+#include "pendulum.h"
```

Comment:
FWIW, I got dinged in #1962 for not including the full path to a file in the current directory...  The google style guide seems to point in that direction with the dir/foo2.h example in https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes